src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -12,23 +14,40 @@
 from __future__ import division, print_function
 
 from pprint import pformat
+from datetime import datetime
 
 # WMCore modules
 from WMCore.MicroService.DataStructs.DefaultStructs import UNMERGED_REPORT
 from WMCore.MicroService.MSCore import MSCore
 from WMCore.MicroService.MSUnmerged.MSUnmergedRSE import MSUnmergedRSE
+from WMCore.Services.RucioConMon.RucioConMon import RucioConMon
+from WMCore.Services.WMStatsServer.WMStatsServer import WMStatsServer
 # from WMCore.Services.AlertManager.AlertManagerAPI import AlertManagerAPI
 from WMCore.WMException import WMException
 from Utils.Pipeline import Pipeline, Functor
 
+import random
+from future import standard_library
```

Comment:
This standard_library change was added to the wrong module. It should have been on the one importing urllib.parse. And please make sure it's right below the `__future__` import.

Reply:
fixed